generate_prometheus_targets.sh

```diff
@@ -45,7 +45,8 @@ for project in mlab-sandbox mlab-staging mlab-oti ; do
     # snmp_exporter on port 9116
     ./mlabconfig.py --format=prom-targets-sites \
         --template_target=s1.{{sitename}}.measurement-lab.org:9116 \
```

Comment:
Actually, I'm realizing from a comment on the relabel config, that the `:9116` annotation here may not be necessary. The snmp exporter port is hard coded in the final relable rule that sets the `snmp-exporter...` `__address__`. 

I think it may be okay to leave the switch undecorated, and to strip off the automatically added port 80 like in the blackbox rules.

Reply:
This is fixed, I think. I've removed the port number from this file, and also removed the check for the port number from the prometheus.yaml config in the prometheus-support repo.